dags/kpi_forecasting.py

```diff
@@ -0,0 +1,93 @@
+"""
+See [kpi-forecasting in the docker-etl repository]
+(https://github.com/mozilla/docker-etl/blob/main/jobs/kpi-forecasting).
+
+This DAG is low priority.
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from utils.gcp import gke_command
+from operators.task_sensor import ExternalTaskCompletedSensor
+from utils.tags import Tag
+
+default_args = {
+    "owner": "pmcmanis@mozilla.com",
+    "email": ["pmcmanis@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2022, 3, 28),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+tags = [Tag.ImpactTier.tier_3]
+
+with DAG("kpi_forecasting", default_args=default_args, schedule_interval="0 4 * * SAT", doc_md=__doc__, tags=tags,) as dag:
```

Comment:
I changed the schedule slightly. The unified_metrics and mobile_usage queries are run at 3am UTC, since the KPI forecasting depends on them we can schedule it after they are likely done (4am).